apps/insights/src/services/clickhouse.ts

```diff
@@ -336,15 +336,47 @@ export const _getPublisherAverageScoreHistory = async ({
     },
   );
 
-// note that this is not cached as the `until` param is a unix timestamp
+type ResolutionUnit = "SECOND" | "MINUTE" | "HOUR" | "DAY";
+type Resolution = `${number} ${ResolutionUnit}`;
+// note that this is not cached as `from` and `to` params are unix timestamps
 export const getHistoricalPrices = async ({
   symbol,
-  until,
+  from,
+  to,
+  publisher = "",
+  resolution = "1 MINUTE",
+  cluster = "pythnet",
 }: {
   symbol: string;
-  until: string;
-}) =>
-  safeQuery(
+  from: number;
+  to: number;
+  publisher: string | undefined;
+  // TODO fhqvst Should we prevent fetching second-resolution data?
+  resolution?: Resolution;
+  cluster?: "pythnet" | "pythtest";
+}) => {
+  const queryParams: Record<string, number | string | string[]> = {
+    symbol,
+    from,
+    to,
+    publisher,
+  };
+
+  const clusterClause =
+    cluster === "pythtest"
+      ? "cluster IN {clusters: Array(String)}"
+      : "cluster = {cluster: String}";
+
+  if (publisher) {
+    queryParams.publisher = publisher;
+  }
+  if (cluster === "pythtest") {
+    queryParams.clusters = ["pythnet", "pythtest-conformance"];
```

Comment:
I'm confused why we would fetch from both clusters when pythtest is specified?  It also does not look like the cluster is ever passed in so why even have this as a config?

Reply:
so this in an artifact from [here](https://github.com/pyth-network/web-api/blob/main/web_api/db.py#L106-L108), but for some reason i've messed it up (pythnet instead of pythtest)... and tbh i'm not even sure we wanna do this at all.

i'll remove the addition of `pythtest-conformance` so that cluster can only be `pythnet` or `pythtest`, and we can add back any conformance-related stuff later.